dts/arm/st/stm32f4.dtsi

```diff
@@ -135,6 +135,19 @@
 			status = "disabled";
 			label = "SPI_2";
 		};
+
+		otgfs: otgfs@50000000 {
```

Comment:
Can we rename as `usbotg_fs: usb@50000000` ?

Reply:
Fixed.
@erwango Are you ok with the label, compatible, interrupt-names strings?